Kata RPG/Kata RPG/Priest.swift

```diff
@@ -10,22 +10,20 @@ import Foundation
 
 class Priest: Character {
 
-        override func attack(target: Character){
+         func attack(target: Character){
                print("\(self.name) is a priest and can't attack")
             }
 
 
-    override func heal(target: Character) {
-                    if(self.lifeStatus == true){
+        func heal(target: Character) {
+                    if(self.lifeStatus == true && (self.faction === target.faction && self.faction != nil && target.faction != nil ) ){
```

Comment:
Not sure that's what you wanted to do. You should extract a function and name it so we know what it does at first glance